README.md

```diff
@@ -187,8 +160,68 @@ Logical volume size can be extended, but not reduced -- this is for
 safety, as manual intervention is probably required for data
 migration, etc.
 
-Contributors
-=======
+## Type Documentation
```

Comment:
Could you move the type documentation to the reference section? This should come before the limitations section, as per https://docs.puppetlabs.com/puppet/latest/reference/modules_documentation.html